### ISSUE TITLE
ios open pre-populated db from file problem

### ISSUE THREAD

**Issue by sinodko** [andpor/react-native-sqlite-storage#311]
I am not able to open pre-populated db file on ios however, I have no problem with opening the same file on android device.

## Expected Behavior

open pre-populated db file on ios.

## Current Behavior

After calling openDatabase, the database file should be opened instead of creating an empty database

## Steps to Reproduce (for bugs)

My project structure looks like

- MyProject
  - android
  - ios
    - MyProject
      - www
        - db.sqlite
    - ...
  - src
    - index.js
    - database_context.js
 
And my code in `database_context.js`:

    SQLite.openDatabase({
        name: "db.sqlite",
        createFromLocation:  1,
        location: "Documents",
        readOnly: true
      }, db => {this.setState({db})}
    );

Every time the openDatabase command is called, it is creating a new databse instead of importing the pre-populated one, which is located as you can see in my project structure. Can anyone help me please?

Note: On Android device, my db file is located `android/app/src/main/assets/www` folder and everything works great.

EDIT: I also tried to move `www/` folder to the root of project and to the `src/` folder but nothing seems to be working.

**Comment by andpor** [comment]
on ios try this:

```
SQLite.openDatabase({
    name: "db.sqlite",
    createFromLocation : 1
    readOnly: true
  }, db => {this.setState({db})}
);
```

refer to Readme for [instructions](https://github.com/andpor/react-native-sqlite-storage/blob/master/README.md#step-6---adjust-opendatabase-call) to

**Comment by andpor** [comment]
I suggest you go through the XCode IDE debug console message and see what is happening on the native side...

**Comment by sinodko** [comment]
I did not :/

**Comment by jaysbytes** [comment]
I've experienced a similar issue. I successfully opened and queried a database on android, but on ios the database seems to be opened but a table with data was not inside. Also, a query to `sqlite_master` returned an empty result (`SELECT * FROM sqlite_master`). Then I tried to run the app through XCode, and app stared querying a table with success. After that, building an app using `react-native run-ios` still does not give a positive result :/. I'm not sure if it will work on a real device or not.
